### PR TITLE
Updates to site

### PR DESCRIPTION
**Pull request by ed1d1a8d**

### CODE REVIEW

File: client/react/Navbar.js [hunk 1]
```diff
@@ -1,33 +1,61 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 
 class Navbar extends Component {
   render() {
     return (
       <nav className="navbar navbar-static-top" role="navigation">
         <div className="container-fluid">
           <div className="navbar-header">
-            <button className="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
```
Comment (Firescar96) [inline client/react/Navbar.js]: are you sure you want to remove this? This is supposed to collapse on mobile to make an expandable menu, ofc if it doesn't work that's a bug.

File: client/react/Macweb.js [hunk 2]
```diff
@@ -8,15 +8,15 @@ class Macweb extends Component {
     //Image map created by jtwright
     let map = (
       <map name="macg">
-        <area href="/entry-pages/a-entry.pdf" shape="poly" coords="285,68,329,92,417,42,417,93,329,144,285,119" alt="Link" data-maphilight='{"strokeColor":"ccf8ff","strokeWidth":4,"fillColor":"ccf8ff","fillOpacity":1}' title="A Entry" />
```
Comment (Firescar96) [inline client/react/Macweb.js]: these colors weren't chosen arbitrarily, every entry IRL is actually trimmed with a different color paint. I used those colors as the hex values, then some entries wanted to be eg. "Pirate Black" so I let changed it. Up to you, though.

File: client/react/Housecomm.js [hunk 3]
```diff
@@ -38,8 +38,8 @@ class Housecomm extends TrackerReact(Component) {
                     <thead>
                       <tr>
                         <th>Position</th>
-                        <th>Person</th>
-                        <th>Email</th>
+                        <th>Person(s)</th>
+                        <th>Kerberos Username(s)</th>
```
Comment (Firescar96) [inline client/react/Housecomm.js]: non-mit people (or prefrosh) who use the site aren't going to know what a kerberos is, rare, but google analytics shows that it happens. this needs some kind of explainer.

File: client/react/Navbar.js [hunk 4]
```diff
@@ -40,6 +40,7 @@ class Navbar extends Component {
             <li id="nav-laundry">
               <a
                 href="http://laundryview.com/laundry_room.php?lr=1364815"
+                target="_blank"
```
Comment (Firescar96) [inline client/react/Navbar.js]: still opening in the same window

File: client/react/Navbar.js [hunk 5]
```diff
@@ -1,33 +1,61 @@
...
-              <li id="nav-desk"><a href="/desk">Movie List</a></li>
-              <li id="nav-laundry"><a href="http://laundryview.com/laundry_room.php?lr=1364815">Laundry</a></li>
-              <li id="nav-dininghall"><a href="/dininghall">Dining Hall</a></li>
-              <li id="nav-about"><a href="/about">About</a></li>
-              <li><a href='https://github.com/Firescar96/macweb' id='github' className='fa fa-github'>Report Issue</a></li>
-            </ul>
-          </div>
+          <ul className="nav navbar-nav">
+            <li id="nav-housecomm">
+              <a href="/housecomm" style={{ outline: 0 }}>
```
Comment (Firescar96) [inline client/react/Navbar.js]: https://www.thoughtco.com/g00/avoid-inline-styles-for-css-3466846?i10c.encReferrer=aHR0cHM6Ly9kdWNrZHVja2dvLmNvbS8%3D
external styles are your friend

File: .meteor/packages [hunk 6]
```diff
@@ -23,7 +23,8 @@ andruschka:bootstrap-image-gallery
 kadira:flow-router
 momentjs:moment
 themeteorchef:jquery-validation
-twbs:bootstrap
 ultimatejs:tracker-react
 rzymek:fullcalendar
 shell-server@0.2.3
+fortawesome:fontawesome
```
Comment (Firescar96) [inline .meteor/packages]: nice

File: client/react/Navbar.js (hunk 5 above)
Comment (ed1d1a8d) [reply client/react/Navbar.js]: I see, made it an issue: https://github.com/Firescar96/macweb/issues/20

File: client/react/Navbar.js (hunk 4 above)
Comment (ed1d1a8d) [reply client/react/Navbar.js]: Hmm. Do you know how to fix this? I got the solution from here: https://stackoverflow.com/questions/17711146/how-to-open-link-in-new-tab-on-html

File: client/react/Navbar.js (hunk 1 above)
Comment (ed1d1a8d) [reply client/react/Navbar.js]: Yeah it wasn't working on mobile for me. I opened an issue about fixing the collapsible navbar.

File: client/react/Macweb.js (hunk 2 above)
Comment (ed1d1a8d) [reply client/react/Macweb.js]: I see. I did not realize this was the case. I will change the colors back to what they were -- though keeping the current saturation/lightness.

File: client/react/Housecomm.js (hunk 3 above)
Comment (ed1d1a8d) [reply client/react/Housecomm.js]: Will add explanatory statement.

**Pull request merged**